agent/websockets/shim.go

```diff
@@ -132,26 +140,30 @@ const (
         if (typeof msg == 'string') {
           return false;
         }
-        if (Array.isArray(msg)) {
-          if (msg.length == 1) {
-            if (typeof msg[0] == 'string') {
-              return false;
-            }
-          }
-        }
-        return true;
+		if (Array.isArray(msg)) {
+		  if (msg.length == 1) {
+	  		if (typeof msg[0] == 'string') {
+  			  return false;
+			}
+		  }
+		}
+		return true;
```

Comment:
Somehow the whitespace got mangled here (maybe an issue with spaces being replaced with tabs?)

Reply:
Fixed